app/assets/javascripts/kuroko2/instance_linker.js

```diff
@@ -1,10 +1,25 @@
 jQuery(function ($) {
-  $('td.log').each(function(){
-    var logText = $(this).html();
-    $(this).html(
-      logText.replace(/instance#(\d+)\/(\d+)(?:\s+|\.)/, function(match, jobId, instanceId){
-	return '<a href="/definitions/' + jobId + '/instances/' + instanceId+ '">' + match + '</a>';
-      })
-    );
+  var MutationObserver = window.MutationObserver || window.WebKitMutationObserver || window.MozMutationObserver;
+  if (!MutationObserver) {
+    return;
```

Comment:
Thank you for your information.
To support PhantomJS 2.0 in https://github.com/cookpad/kuroko2/pull/70 .
Could you add a test after merged #70 ?